containers/init/ready/ready.go

```diff
@@ -269,4 +278,19 @@ func main() {
 	log.Printf("all pods ready, exiting successfully")
 	workerFileBody := strings.Join(podIPs, ",")
 	ioutil.WriteFile(outputFile, []byte(workerFileBody), 0777)
+
+	test, err := grpcClientset.LoadTestV1().LoadTests(corev1.NamespaceDefault).Get(os.Args[1], metav1.GetOptions{})
+	if err != nil {
+		log.Fatalf("failed to fetch loadtest for obtaining metadata: %v", err)
+	}
+
+	outputMetadataFile := DefaultMetadataOutputFile
+	outputMetadataFileOverride, ok := os.LookupEnv(OutputMetadataEnv)
+	if ok {
+		outputMetadataFile = outputMetadataFileOverride
+	}
+
+	metaDataSet := test.ObjectMeta
+	metaDataBody, _ := json.Marshal(metaDataSet)
```

Comment:
Would it make sense to check for an error here?

Reply:
Right, I will add a check here.